docs/src/components/pricing/PricingList.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { useTheme } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+import Container from '@material-ui/core/Container';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import KeyboardArrowRightRounded from '@material-ui/icons/KeyboardArrowRightRounded';
+import Link from 'docs/src/modules/components/Link';
+import { PlanName, PlanPrice } from 'docs/src/components/pricing/PricingTable';
+
+function Plan({
+  plan,
+  benefits,
+  unavailable,
+}: {
+  plan: 'community' | 'pro' | 'premium';
+  benefits?: Array<string>;
+  unavailable?: boolean;
+}) {
+  const globalTheme = useTheme();
+  const mode = globalTheme.palette.mode;
+  return (
+    <Paper
+      variant="outlined"
+      sx={{ p: 2, ...(unavailable && { '& .MuiTypography-root': { opacity: 0.2 } }) }}
+    >
+      <PlanName plan={plan} />
+      <Box {...(plan === 'community' && { my: 2 })} {...(plan === 'premium' && { mb: 2 })}>
+        <PlanPrice plan={plan} />
+      </Box>
+      {unavailable ? (
+        <Button
+          component={Link}
+          noLinkStyle
+          href={plan === 'community' ? '/getting-started/usage/' : '/components/data-grid/'}
+          variant="outlined"
+          disabled
+          fullWidth
+          endIcon={<KeyboardArrowRightRounded />}
+          sx={{ py: 1, '&.Mui-disabled': { color: 'text.secondary' } }}
+        >
+          Available later this year!
+        </Button>
+      ) : (
+        <Button variant="outlined" fullWidth endIcon={<KeyboardArrowRightRounded />} sx={{ py: 1 }}>
+          Get Started
+        </Button>
+      )}
+      {benefits &&
+        benefits.map((text) => (
+          <Box key={text} sx={{ display: 'flex', alignItems: 'center', mt: 2 }}>
+            <img src={`/static/branding/pricing/yes-${mode}.svg`} alt="" />
```

Comment:
I would challenge the fact that we ever compose the image URLs. My concern is that it will make it a lot harder to remove assets, we have no guarantee when grepping that it's not used.

Also, the usage of `/static` is from a constraint of 2018 when I first introduce Next.js. Now, we could evaluate if importing them in JS is more relevant, if we do we wouldn't be able to create URLs dynamically either.

Reply:
I will keep this in backlog for refinement, given it is the implementation detail.